app_flutter/lib/model/task.proto

```diff
@@ -25,4 +25,5 @@ message Task {
     optional string buildNumberList = 16;
     optional string builderName = 17;
     optional string luciBucket = 18;
+    optional bool is_test_flaky = 19;
```

Comment:
Would it make sense to add `bool bringup` to match what's being done in the ci.yaml, and reuse the is_flaky field?

Reply:
It would be good to keep consistent between frontend and backend, but I would prefer these names refreshed in a separate PR to make things easier.
https://github.com/flutter/flutter/issues/86721